### ISSUE TITLE
Weird line?

### ISSUE THREAD

**Issue by whaaaley** [KDCinfo/expired-to-be#1]
I got a weird line in the middle of the descriptive text.

![img](https://cl.ly/3a0V1o041S3t/Screen%20Shot%202018-02-21%20at%202.14.34%20PM.png)

**Comment by KDCinfo** [comment]
Thanks for submitting this. It would appear to be a CSS or font issue between OS's. That line is a Unicode character in a `::before` pseudo-element. 

[popup.html]
`content: '\23A1';/*! Yes: 23A1 23AD 0283 222E 257F 299A 29D8 29D9 |No: 222B 2757*/`

I'm trying to get an environment up to recreate this. I run Windows, and I'm guessing this is on iOS. I got Chrome installed on my iPad, but can't get extensions to load (despite Google searches reflecting it is possible). I got  Chromium installed on my Ubuntu 16.04.3, and it shows up "okay" (could use a tweak).

May take a bit to figure something out for iOS, but I'm determined to recreate this, so I can make additional adjustments based on your screenshot (**thanks for that!**). One option is I'll see if I can't somehow emulate the extension as a regular HTML page or something I can hit from my iPad.

![platforms-x3](https://user-images.githubusercontent.com/495285/36514509-94b09d98-1729-11e8-924c-f614ed0167d0.png)
